desktop/config/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
Everything on `desktop` runs as a standard node.js app, and indeed, it has to be vanilla JS.
I'd like to add a build step in the future to transpile the desktop code to enable some more advanced syntax features. Maybe for `v1.0.1` :smile: